backend/src/openarchiefbeheer/logging/logevent.py

```diff
@@ -23,7 +24,30 @@ def _create_log(
 
 
 def destruction_list_created(destruction_list: DestructionList, user: User) -> None:
-    _create_log(model=destruction_list, event="destruction_list_created", user=user)
+    _create_log(
+        model=destruction_list,
+        event="destruction_list_created",
+        user=user,
+        extra_data={
+            "pk": destruction_list.pk,
+            "name": destruction_list.name,
+            "author": {
+                "pk": destruction_list.author.pk,
+                "email": destruction_list.author.email,
+                "username": destruction_list.author.username,
+            },
+            "assignees": [
+                {
+                    "user": {
+                        "pk": assignee.user.pk,
+                        "email": assignee.user.email,
+                        "username": assignee.user.username,
+                    },
+                }
+                for assignee in destruction_list.assignees.exclude(role=ListRole.author)
+            ],
+        },
```

Comment:
We could use the Assignee serializer for this :smile:

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/175

Reply:
As discussed: interesting idea but we the current serializers provide different information aimed to be used in the context of an API. The log created here provides minimal human-readable reference information. We could write another serializer for this but maybe pick this up in a bigger refactor round: https://github.com/maykinmedia/open-archiefbeheer/issues/175?